benchmarks/replay/replay.cpp

```diff
@@ -403,7 +403,7 @@ int main(int argc, char** argv)
     auto const num_threads = per_thread_events.size();
 
     // Uncomment to enable / change default log level
-    // rmm::logger().set_level(rmm::level_enum::trace);
+    // rmm::logger().set_level(rapids_logger::level_enum::trace);
```

Comment:
Would this require an `#include` of some `rapids_logger` header?

Reply:
To be properly IWYU, yes, although in practice including the `rmm/logger.hpp` header will bring in the `rapids_logger` symbols transitively.